{"name": "Проверка текста ошибки при попытке создания курьера с уже существующим логином", "status": "passed", "start": 1747402267533, "stop": 1747402267712, "uuid": "d001fa4c-6fab-46ac-a6d9-6dd057377bf2", "historyId": "b37c628ed87888dcad1b505c1d68ad90", "testCaseId": "b37c628ed87888dcad1b505c1d68ad90", "fullName": "tests.couriers.test_create_courier.TestCreateCourier#test_create_same_courier_error_message", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Pro-Mikhail.local"}, {"name": "thread", "value": "32108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_create_courier"}]}